src/index.html

```diff
@@ -7,12 +7,54 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+
+  <body class="page">
+    <section
+      data-qa="card"
+      class="card"
+    >
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="Macbook"
+      />
+
+      <h2 class="card__description">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__reviews">
+        <div class="card__stars stars">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </div>
+
+        <p class="card__reviews-reviews">Reviews: 5</p>
+      </div>
+      <div class="card__price">
+        <p class="card__price-text">Price:</p>
+        <p class="card__price-value">$2,199</p>
+      </div>
+      <button
+        data-qa="hover"
+        class="card__button"
+      >
+        Buy
+      </button>
```

Comment:
Consider adding a type attribute to the button element. Although the default type of a button is 'submit', it's a good practice to always specify it to avoid unexpected behavior in different browsers.